recipe/meta.yaml

```
@@ -18,14 +26,24 @@
   {% set cuda_minor = 0 %}
 {% endif %}
 
+# baseline for build number for existing subpackages to align build string with build number and correct priority
+# TODO: next release, set baseline to 0 or remove variable
+{% set build_num_baseline = 200 %}
```

```suggestion
{% set build_num_baseline = 0 %}
```
This variable can be removed *now* because due to a bug in the previous recipe:

```yaml
    build:
      number: {{ build_num }}
```
The build number was never offset for MPI variants. Instead, all package variants had a build number of 0 which will be less priority than 1 and 101.

---

Thanks! I removed it from the recipe.